.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
```

Comment:
I would suggest changing to version `2.1` here.

Reply:
Done.